gnomad/resources/grch38/reference_data.py

```diff
@@ -383,3 +383,38 @@ def get_truth_ht() -> Table:
         .repartition(200, shuffle=False)
         .persist()
     )
+
+
+def _import_gencode(gtf_path: str, **kwargs) -> hl.Table:
```

Comment:
Is this identical to the one already in grch37? If so, instead of having the copy and paste, do you think we should have it in `resource_utils.py`?

Reply:
Yes, it is, I just moved to resource_utils.py and changed both reference_data.py for both grch37 and grch38.